config.yml.example

```diff
@@ -157,7 +157,7 @@
 #
 #
 ##  Enable usage of Connectors API instead of calling connectors indices directly
```

Comment:
I really liked @seanstory's addition to the flag description in [this closed PR](https://github.com/elastic/connectors/pull/2898/files). I think we should also add this.

```suggestion
##  Enable usage of Connectors API instead of calling connectors indices directly
##    Using direct index access is deprecated, and will be disallowed entirely in a future version
```